{"name": "test_expected_failures", "status": "skipped", "statusDetails": {"message": "XFAIL reason: 该功能尚未完善\n\n_pytest.outcomes.XFailed: 该功能尚未完善", "trace": "self = <test1--各种装饰器学习.Test_1 object at 0x000001F197D38788>\n\n    def test_expected_failures(self): #对应测试报告里面的expected failures--期望失败\n>       pytest.xfail(reason='该功能尚未完善')\nE       _pytest.outcomes.XFailed: 该功能尚未完善\n\ntestcase\\test1--各种装饰器学习.py:141: XFailed"}, "start": 1683685854072, "stop": 1683685854072, "uuid": "8a734884-1acc-41f6-a941-6652794fc77d", "historyId": "3fbe20abd5a0aa11c4a9891b08858c13", "testCaseId": "860d21a28a7497034096adc1827a6163", "fullName": "testcase.test1--各种装饰器学习.Test_1#test_expected_failures", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test1--各种装饰器学习"}, {"name": "subSuite", "value": "Test_1"}, {"name": "host", "value": "LAPTOP-8ERGQRFP"}, {"name": "thread", "value": "8948-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test1--各种装饰器学习"}]}